pkg/capacityscheduling/capacity_scheduling.go

```diff
@@ -192,15 +192,15 @@ func New(ctx context.Context, obj runtime.Object, handle framework.Handle) (fram
 	return c, nil
 }
 
-func (c *CapacityScheduling) EventsToRegister() []framework.ClusterEventWithHint {
+func (c *CapacityScheduling) EventsToRegister(_ context.Context) ([]framework.ClusterEventWithHint, error) {
```

Comment:
```suggestion
func (c *CapacityScheduling) EventsToRegister(context.Context) ([]framework.ClusterEventWithHint, error) {
```

I'm not sure that this repository coding policy. But, `_` seems to be redundant.

Reply:
Both is fine I think. i will leave it as-is which is consistent with the upstream coding style.

Reply:
That makes sense.